build/azuredevops-runner.yml

```diff
@@ -118,6 +127,15 @@ stages:
               wget https://github.com/taskctl/taskctl/releases/download/${{ variables.TaskctlVersion }}/taskctl_${{ variables.TaskctlVersion }}_linux_amd64.tar.gz -O /tmp/taskctl.tar.gz
               tar zxf /tmp/taskctl.tar.gz -C /usr/local/bin taskctl 
 
+        # Install Stacks Envfile to generate the envfile to be used by docker
+        - task: Bash@3
+          displayName: Install Envfile
+          inputs:
+            targetType: inline
+            script: |
+              wget https://github.com/amido/stacks-envfile/releases/download/v${{ variables.StacksEnvfileVersion }}/stacks-envfile-linux-amd64-${{ variables.StacksEnvfileVersion }} -O /usr/local/bin/envfile
+              chmod +x /usr/local/bin/envfile
+
```

Comment:
Are we installing Envfile twice?

Reply:
Yeah we are, my bad - thought we needed to but of course it is all in the same agent, will fix that one

Reply:
As this is a differennt stage in the pipeline it is run in a different session so the tools that were installed during the build stage are not available, this means that `taskctl` and `envfile` need to be installed int he Release stage as well